api/impots/impots.controller.js

```diff
@@ -57,7 +57,7 @@ function ImpotController (options) {
     var numeroFiscal = options.numeroFiscal
     var referenceAvis = options.referenceAvis
     if (!numeroFiscal || !referenceAvis) {
-      return next(new StandardError('Les paramètres numeroFiscal et referenceAvis doivent être fournis dans la requête.', {code: 400, scope: 'dgfip'}))
+      return next(new StandardError('Les paramètres numeroFiscal et referenceAvis doivent être fournis dans la requête.', {code: 400}))
```

Comment:
I don't get it. Can you explain this to me ?

Reply:
lorsque tu ajoutes un scope à ton erreur, la réponse est filtrée pour savoir si tu es ou pas autorisé à voir l'erreur